src/main/java/ch/njol/skript/expressions/ExprFormatDate.java

```diff
@@ -92,7 +123,7 @@ public Class<? extends String> getReturnType() {
 
 	@Override
 	public String toString(@Nullable Event e, boolean debug) {
-		return getExpr().toString(e, debug) + " formatted as " + format.toPattern();
+		return getExpr().toString(e, debug) + " formatted as " + (customFormat != null ? customFormat.toString(e, debug) : DEFAULT_FORMAT.toPattern());
```

Comment:
Does this need to account for `format` if a user uses that? AKA try to use `format` before `DEFAULT_FORMAT`